### ISSUE TITLE
Upgraded to new phone but nightwatch wont work! 

### ISSUE THREAD

**Issue by j3nl** [StephenBlackWasAlreadyTaken/NightWatch#130]
Hi I changed from an s9 to an s10. Nightwatch worked brilliantly on my s9 and watch but since changing across nightwatch is stuck on the last reading from my s9 and will no longer receive readings from dexcom? I've tried deleting the app and reinstalling, logging out etc but no joy! Please help me get it back!!

**Comment by thanksegon** [comment]
Did you delete the app from both phone and watch, and then reboot?


Joe Ritson
Cell Phone: 617.775.7750
LinkedIn: www.linkedin.com/in/josephritson<https://nam05.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.linkedin.com%2Fin%2Fjosephritson&data=02%7C01%7C%7C0850151ed34b4c77494208d80eecebec%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637275758010462860&sdata=rf7LUtVoHgFV5vy8NVGLMGXbHR1hVPHwXduJBPXliQY%3D&reserved=0>

________________________________

**Comment by j3nl** [comment]
I deleted both from phone and watch. Do you mean then turn them both off and back on again or just reload the app to both?

**Comment by thanksegon** [comment]
I'm just grasping but uninstall and reboot both before installing again.


Joe Ritson
Cell Phone: 617.775.7750
LinkedIn: www.linkedin.com/in/josephritson<https://nam05.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.linkedin.com%2Fin%2Fjosephritson&data=02%7C01%7C%7C0850151ed34b4c77494208d80eecebec%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637275758010462860&sdata=rf7LUtVoHgFV5vy8NVGLMGXbHR1hVPHwXduJBPXliQY%3D&reserved=0>

________________________________

**Comment by j3nl** [comment]
OK I'll give it a go and see, I'll let you know haha!

**Comment by j3nl** [comment]
No Joy 😔, any other ideas?

**Comment by thanksegon** [comment]
No. Nightwatch hasn't had an update in 4 years, I think, so it's a wonder if still works at all. Sorry I can't help.


Joe Ritson
Cell Phone: 617.775.7750
LinkedIn: www.linkedin.com/in/josephritson<https://nam05.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.linkedin.com%2Fin%2Fjosephritson&data=02%7C01%7C%7C0850151ed34b4c77494208d80eecebec%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637275758010462860&sdata=rf7LUtVoHgFV5vy8NVGLMGXbHR1hVPHwXduJBPXliQY%3D&reserved=0>

________________________________

**Comment by psonnera** [comment]
@StephenBlackWasAlreadyTaken the app is stale for some time now, should we recommend users to move to xDrip+ or would it make sense to try to upgrade it and recompile it to support recent Android versions? Thanks.

**Comment by j3nl** [comment]
Personally I'd love it if you were to upgrade it because it worked beautifully before! It was a great piece of IT genius haha!

**Comment by thanksegon** [comment]
Can xdrip be used to follow?

Personally, I like NW simplicity, and its alarm options, like xdrip, are better than anything else out there. I'm not a developer but I'd say the ui is fine, minus a few bugs that need to be addressed.

Joe Ritson
Cell Phone: 617.775.7750
LinkedIn: www.linkedin.com/in/josephritson<https://nam05.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.linkedin.com%2Fin%2Fjosephritson&data=02%7C01%7C%7C0850151ed34b4c77494208d80eecebec%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637275758010462860&sdata=rf7LUtVoHgFV5vy8NVGLMGXbHR1hVPHwXduJBPXliQY%3D&reserved=0>


________________________________

**Comment by psonnera** [comment]
@thanksegon xDrip+ yes sure! Take the hardware data source Nightscout Follower.

**Comment by psonnera** [comment]
> @StephenBlackWasAlreadyTaken the app is stale for some time now, should we recommend users to move to xDrip+ or would it make sense to try to upgrade it and recompile it to support recent Android versions? Thanks.

@AdrianLxM what do you think?

**Comment by AdrianLxM** [comment]
As a Viewer for Nightscout we could revive it.
I have no intention to (and time) to work with share.

**Comment by keyanooshgal** [comment]
I've updated to the S21 and Nightwatch no longer updates blood sugar numbers unless I log out and then back in, so I'm back to following my son on Dexcom Follow. I would be so happy to see an update that would make NW usable again!

**Comment by thanksegon** [comment]
You can download xdrip and it will connect with NS in the same way as nightwatch. There are watch faces, alarms, etc. Nightwatch is a dead product.


Joe Ritson
Cell Phone: 617.775.7750
LinkedIn: www.linkedin.com/in/josephritson<https://nam05.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.linkedin.com%2Fin%2Fjosephritson&data=02%7C01%7C%7C0850151ed34b4c77494208d80eecebec%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637275758010462860&sdata=rf7LUtVoHgFV5vy8NVGLMGXbHR1hVPHwXduJBPXliQY%3D&reserved=0>

________________________________

**Comment by keyanooshgal** [comment]
I'm just following my son though. Would he have to have x drip or could I
use x drip and get his info from Dexcom with his user name and password?
I have xdrip downloaded but haven't set it up because it seems like I have
to be the one with the Dexcom and it's my son. I could possibly convince
him to set it up on his phone but if there's anyway to just follow him with
xdrip, I'd love it! I'm not sure about Azure and all of the other
components to having xdrip work?
Thanks for responding so quickly!
-Keyanoosh
